requirements.in

```diff
@@ -9,9 +9,11 @@ cryptography
 cyclone
 datadog
 gcm-client
+firebase-admin
 hyper
 marshmallow
 marshmallow-polyfield
+oauth2client
```

Comment:
the latest firebase-admin switched from [oauth2client to google-auth](https://firebase.google.com/docs/admin/migrate-auth-50#dependency_on_oauth2client_replaced_with_google-auth) but we still use it (in a compatible way AFAICT)